examples/summarize/src/summarize.py

```diff
@@ -0,0 +1,52 @@
+import os
+from airtop import Airtop, SessionConfigV1
+from dotenv import load_dotenv
+
+load_dotenv()
+
+# Setup environment variables
+api_key = os.getenv("AIRTOP_API_KEY")
+if not api_key:
+    print("Error: AIRTOP_API_KEY environment variable must be set.")
+    exit(1)
+
+TARGET_URL = "https://www.nba.com"
```

Comment:
This URL is just returning site security info:
![image](https://github.com/user-attachments/assets/41f59f59-fdae-4a52-b89c-ef0843b4eda8)

Let's use the same URL as the TS version of this example:
```suggestion
TARGET_URL = "https://en.wikipedia.org/wiki/A.I._Artificial_Intelligence"
```